test/device/test_accelerometer.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+from app.device.accelerometer import Accelerometer
+
+
+def test_not_moving():
+#sample data while not moving
+    raw_data = [18, 0, 4, 0, 234, 0]

```

Comment:
shouldnt these be bytes not longs?

Reply:
bus.read_i2c_block_data returns a list of 6 integers:

>>> bus.read_i2c_block_data(ADDRESS, DATAX0, 6)
>>> [115, 0, 0, 0, 0, 29]

Reply:
data is read from the sensor as an array of integers